tests/test_bencode.py

```diff
@@ -99,3 +99,9 @@ def test_encode_complex():
 def test_encode_errors():
     with pytest.raises(BencodeEncodingError):
         Bencode.encode(object())
+
+def test_encode_error_long():
+    # os.path.getsize() can be longs on py2
+    if sys.version_info <= (3, 0):
+        a_long = 741634835L
+        encoded_long = Bencode.encode(a_long)
```

Comment:
I see. Here we should use @pytest.mark.skipif so to check python version, otherwise we get SyntaxError on Py3. https://travis-ci.org/idlesign/torrentool/jobs/157047830#L260